lib/Room.js

```diff
@@ -816,6 +816,74 @@ class Room extends EventEmitter
 				break;
 			}
 
+			case 'getPeers':
```

Comment:
When reading the client code I was pretty confused that the result of this wasnt being used. I did not realize it actually ended up creating consumers. Maybe we should name it something like `consumeExistingProducers`? We might also want oto use this on joining rather than doing that automatically.

Alternatively, it might be nice to actually just return the list of existing producing peers and have the client request to create producers. (both in this case, and the join case)... Though that might be better for another PR.

Reply:
I'm going to refactor existing producers common code but I think it's ok leaving the flow as it is where consumers for existing producers are returned inline, otherwise it's another message that we have to exchange and traverse again the whole peers map for something that we always are going to need to to in both cases.

Maybe I up change the method name to `refreshConsumers`. The returned peers are used in the join to hold the connection flow until all consumers are received but in this case it doesn't really makes much sense.